template/install.rdf

```diff
@@ -18,7 +18,12 @@
 
     <!-- Front End MetaData -->
     <em:name>Valence</em:name>
-    <em:description>Protocol Adapters for Firefox Developer Tools</em:description>
+    <em:description>Protocol Adapters for Firefox Developer Tools. User documentation is available on MDN: https://developer.mozilla.org/docs/Tools/Valence
```

Comment:
Yep, I've asked the desktop team about it and they said that I was welcome to fix bug 624602 :)